frontend/src/tests/lib/components/ui/IdentifierHash.spec.ts

```diff
@@ -25,4 +28,17 @@ describe("IdentifierHash", () => {
       `Copy to clipboard: ${identifier}`
     );
   });
+
+  it("should use a unique tooltip ID each time", async () => {
+    const po1 = renderComponent({ identifier });
+    const po2 = renderComponent({ identifier });
```

Comment:
Would this be more realistic?

```suggestion
    const po2 = renderComponent({ identifier: "something else" });
```

Reply:
Not necessarily. I considered it, but another implementation I considered had the ID depend on the identifier, which would actually have failed this test.
At the same time I can imagine having the same identifier in different locations on the page.